portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,86 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Joel's Portfolio 😃</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
```

Comment:
Might be better to move those <style> tags to style.css file.
 Up to you. It's OK to leave them here if only one files uses them.